ci-automation/vendor-testing/qemu_update.sh

```diff
@@ -73,8 +78,30 @@ kola run \
     --platform=qemu \
     --qemu-bios="${bios}" \
     --qemu-image=tmp/flatcar_production_image_previous.bin \
-    --tapfile="${CIA_TAPFILE}" \
+    --tapfile="${CIA_TAPFILE}.previous" \
     --torcx-manifest="${CIA_TORCX_MANIFEST}" \
     --update-payload=tmp/flatcar_test_update.gz \
     --qemu-skip-mangle \
     cl.update.payload
+
+kola run \
+    --board="${CIA_ARCH}-usr" \
+    --parallel="${QEMU_PARALLEL}" \
+    --platform=qemu \
+    --qemu-bios="${bios}" \
+    --qemu-image=tmp/flatcar_production_image_first_dual.bin \
+    --tapfile="${CIA_TAPFILE}.first_dual" \
+    --torcx-manifest="${CIA_TORCX_MANIFEST}" \
+    --update-payload=tmp/flatcar_test_update.gz \
+    --qemu-skip-mangle \
+    cl.update.payload
+
+# When one of the above failed, the whole tests gets rerun
```

Comment:
I wouldn't say it's true. I noticed that `kola run` is executed under `set -e` so any of them failing would result in "${CIA_TAPFILE}" not being created, and thus the test runner not even trying to rerun the failed tests (because the tap analyser expects the file to exist).

Reply:
Yes, it would need manual restarts. I will look into your suggestion to make it work with automatic retries.